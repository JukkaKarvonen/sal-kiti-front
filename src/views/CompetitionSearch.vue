<template>
  <div class="search">
    <CompetitionSearch />
  </div>
</template>

<script>
/**
 * Competition search view
 */
import CompetitionSearch from "@/components/CompetitionSearch.vue";

export default {
  name: "CompetitionSearchView",
  components: {
    CompetitionSearch
  },
  created() {
    document.title = this.$t("title.search") + " | " + this.$t("title.prefix");
  }
};
</script>
