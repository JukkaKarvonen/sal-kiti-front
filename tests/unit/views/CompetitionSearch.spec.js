import { shallowMount, createLocalVue } from "@vue/test-utils";
import CompetitionSearch from "@/views/CompetitionSearch.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("CompetitionSearch.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(CompetitionSearch, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
