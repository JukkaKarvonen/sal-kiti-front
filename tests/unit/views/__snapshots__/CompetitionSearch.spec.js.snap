// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompetitionSearch.vue renders content 1`] = `
<div
  class="search"
>
  <competitionsearch-stub />
</div>
`;
